src/routes/expenses.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
```

Comment:
Consider removing the 'use strict' directive, as it's not necessary in ES6 modules.